packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -49,8 +53,8 @@ export const potentialFirstLoadInDevWhenUATMissing: InterstitialRule = async opt
   return undefined;
 };
 
-export const potentialRequestAfterSignInOrOurFromClerkHostedUiInDev: InterstitialRule = async options => {
-  const { apiKey, secretKey, referrer, host, forwardedHost, forwardedPort, forwardedProto } = options as any;
+export const potentialRequestAfterSignInOrOurFromClerkHostedUiInDev: InterstitialRule = options => {
```

Comment:
@panteliselef isn't this a typo in the function name? `...InOrOutFrom...` instead of `...InOrOurFrom...`

Reply:
Yes it seems like it is. @dimkl can you verify ?

Reply:
yes you are correct, it's a typo.